achievements/imTheDevilILoveMetal.achievement.js

```diff
@@ -0,0 +1,29 @@
+var imTheDevilILoveMetal = {
+  name: 'I\'m the Devil I Love Metal',
+  check: function(pullRequest, shall) {
+    if (isSequenceOfSixes(pullRequest.number)) {
+
+      var achievement = {
+        avatar: 'images/achievements/iAmTheDevilILikeMetal.achievement.gif',
+        name: 'I\'m the Devil I Love Metal',
+        short: 'check this riff it\'s fucking tasty',
+        description: [
+          'Your pull request number ' + pullRequest.number,
+          ' summoned the devil himself. ',
+          'I hope that code you submitted was a masterpiece, ',
+          'or you\'re gonna gargle mayonnaise'
+          ].join(''),
+        relatedPullRequest: pullRequest.id
+      };
+
+      shall.grant(pullRequest.creator.username, achievement);
+    }
+  }
+};
+
+function isSequenceOfSixes(number) {
+  const sequenceOfSixesRegEx = new RegExp(/^6{2,}$/);
```

Comment:
Very small nit :-)

This will totally work, but this is what I suggest instead:
```suggestion
  const sequenceOfSixesRegEx = /^66+$/;
```
Although your regex syntax gives the same result, it is less clear to read as it is right now.
- I think having the regex set as `/^66+$/` makes more sense when you read it, and eventually uses fewer characters altogether :-)
- Also, removing the `new RegExp` won't hurt us, leaves us with less code, and IMO it's more readable. Currently, you define a regex and pass it to the `new RegExp` constructor. The `new RegExp` constructor is useful if you want to build a dynamic regex instead of creating a static regular expression, but that benefit requires the `new RegExp` function to be called with a string (so you can concatinate the result) like `new RegExp('^6{' + minimalRepeat + ',}$')`.

Current implementation uses **51 characters** compared to my suggestion which uses **37 characters** while **performing the same thing**.

Also, I think your can add the missing `p` to the end of the variable name (not sure if you left it out by mistake): ` sequenceOfSixesRegEx` ▶️ `sequenceOfSixesRegExp`

Reply:
Thanks, Implemented.